main.go

```diff
@@ -8,7 +8,6 @@ import (
 	"github.com/wailsapp/wails/v2/pkg/options/assetserver"
 )
 
-//go:embed all:frontend/dist
```

Comment:
I'm shocked this works without the comment here. I guess wails is handling that automatically?